### PR TITLE
Bug 1799540: Node exporter init collectors must be privileged

### PR DESCRIPTION
**Pull request by smarterclayton**
dmidecode (used by the VM type detection) requires access to
/dev/mem which requires privileged. Only the init container gets
that permission. Blocks being able to read physical machine BIOS
info.

### CODE REVIEW

**Comment by LiliC** [comment]
Generate should be fixed by https://github.com/openshift/cluster-monitoring-operator/pull/649

File: assets/node-exporter/daemonset.yaml [hunk 1]
```diff
@@ -87,6 +87,7 @@ spec:
         name: init-textfile
         resources: {}
         securityContext:
+          privileged: true
```
Comment (LiliC) [inline assets/node-exporter/daemonset.yaml]: Think you are missing `allowPrivilegedContainer: true` as well.

**Comment by bparees** [comment]
skipping this one for approval this week due to bug severity and the number of other changes already accepted.

**Pull request merged**